src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/ArrayType.cs

```diff
@@ -20,7 +20,10 @@ internal class ArrayType : CType
         public CType GetBaseElementType()
         {
             CType type;
-            for (type = GetElementType(); type.IsArrayType(); type = type.AsArrayType().GetElementType()) ;
+            for (type = GetElementType(); type.IsArrayType(); type = type.AsArrayType().GetElementType())
+            {
+            }
```

Comment:
The double `; ;` are still worth tidying I suppose, if very minor. I'll bring this back to just removing those.